docs/appendices/release-notes/5.0.0.rst

```diff
@@ -70,17 +70,17 @@ Breaking Changes
   name as column name unless the function returns more than one column. Some
   examples:
```

Comment:
shy not use `::` and then again just before the next `SELECT`?

Reply:
Hi Marios,

reStructuredText rendering, as defined by docutils, is a bit sensitive.

In this specific case, using [literal blocks](https://docutils.sourceforge.io/docs/user/rst/quickref.html#literal-blocks) within [bullet lists](https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists) and friends will cause the list items to be spaced apart by an additional newline, for the _whole_ list. That looks a bit unpleasant to the eye, when other lists are displayed on the same page where this is not happening.

With kind regards,
Andreas.

Reply:
Thanks for the explanation @amotl !